janusgraph-backend-testutils/src/main/java/org/janusgraph/diskstorage/SimpleScanJob.java

```diff
@@ -269,9 +270,11 @@ public static Configuration getJobConf(List<SliceQuery> queries, Long modulus) {
     }
 
     public static Configuration getJobConf(List<SliceQuery> queries, Long modulus, Long modVal) {
+        BaseConfiguration baseConfiguration = new BaseConfiguration();
+        baseConfiguration.setListDelimiterHandler(new DefaultListDelimiterHandler(','));
```

Comment:
Why we need this setting?

Reply:
To maintain backward compatibility. In the old version of commons-configuration, comma-separated values are parsed into a list of values, while in the new version (configuration2), commas are read as they are.

ref:
1. http://tinkerpop.apache.org/docs/3.5.0-SNAPSHOT/upgrade/#_versions_and_dependencies 
2. https://commons.apache.org/proper/commons-configuration/userguide/upgradeto2_0.html

Reply:
Do we want the old behavior?

Reply:
If we don't do this, then comma-delimited values, e.g. "host1,host2,host3" will be read as a single string, rather than a string array ["host1","host2","host3"] (which is the old behavior).

In theory, we can read it as a single string. When we need it, we split it into a string array. The drawback is, this string splitting might happen many times on runtime and thus inefficient. Another problem is, I am not sure if I could implement that change correctly and not miss any place.